### PR TITLE
Updates from Steamnasium work on Feb 28-29

### PR DESCRIPTION
**Pull request by kirbt**

### CODE REVIEW

File: src/main/deploy/launcher_data.csv [hunk 1]
```diff
@@ -2,7 +2,7 @@
...
 
-0, 75, 1500
-1, 30, 2000
-2, 20, 3000
-5, 10, 3500
\ No newline at end of file
+0.8, 310, 3300
+1.5, 300, 3400
+2, 285, 3500
+6, 270, 6000
```
Comment (KangarooKoala) [inline src/main/deploy/launcher_data.csv]: Are there any particular places on the field that these correspond to?

**Review by KangarooKoala — DISMISSED** [review]
We could add the comment later if we want to merge this in quickly so that the days stay separate.

File: src/main/deploy/launcher_data.csv (hunk 1 above)
Comment (bobtseattle) [reply src/main/deploy/launcher_data.csv]: +1 to using particular spots on the field drive team knows are properly calibrated to shoot from and so programming knows to use to recalibrate if there are robot changes.  Ideally these positions are noted in comments as implied above.

File: src/main/java/frc/team2412/robot/subsystems/DrivebaseSubsystem.java [hunk 2]
```diff
@@ -147,7 +148,7 @@ public DrivebaseSubsystem() {
 
 		// LOW verbosity only sends field position, HIGH sends full drive data, MACHINE sends data
 		// viewable by AdvantageScope
-		SwerveDriveTelemetry.verbosity = TelemetryVerbosity.LOW;
+		SwerveDriveTelemetry.verbosity = TelemetryVerbosity.MACHINE;
```
Comment (bobtseattle) [inline src/main/java/frc/team2412/robot/subsystems/DrivebaseSubsystem.java:151]: Should this also be controlled by debug flag?

**Review by bobtseattle — COMMENTED** [review]
Is it clear to drive team which autos have been fully tested and ready to be used in competition?

File: src/main/java/frc/team2412/robot/subsystems/DrivebaseSubsystem.java (hunk 2 above)
Comment (kirbt) [reply src/main/java/frc/team2412/robot/subsystems/DrivebaseSubsystem.java:151]: shouldnt be a problem as long as we merge the logging pr

**Review by jamesdooley4 — APPROVED** [review]
Approving to provide a baseline before competition

File: src/main/deploy/launcher_data.csv (hunk 1 above)
Comment (jamesdooley4) [reply src/main/deploy/launcher_data.csv]: About a foot away from the subwoofer works well

**Pull request merged**